app/package.json

```diff
@@ -37,7 +37,7 @@
     "electron-window-state": "5.0.3",
     "escape-html": "1.0.3",
     "is-online": "7.0.0",
-    "node-json-db": "0.9.1",
+    "node-json-db": "0.9.2",
```

Comment:
@priyank-p, @vsvipul this is safe, right?

Reply:
This only adds the typings required for typescript. So, shouldn't be any problem. This was the only change as seen from the version history [here](https://github.com/Belphemur/node-json-db/releases).